payments-ui-core/src/main/java/com/stripe/android/ui/core/forms/resources/LpmRepository.kt

```diff
@@ -72,37 +72,6 @@ class LpmRepository constructor(
     private val lpmInitialFormData: LpmInitialFormData = LpmInitialFormData.Instance,
     private val lpmPostConfirmData: LuxePostConfirmActionRepository = LuxePostConfirmActionRepository.Instance
 ) {
-    private val lpmSerializer = LpmSerializer()
-
-    var serverSpecLoadingState: ServerSpecState = ServerSpecState.Uninitialized
-
-    val supportedPaymentMethodTypes: List<String> by lazy {
```

Comment:
We don’t need this explicit list. Instead, we should return `null` in `convertToSupportedPaymentMethod()` if a payment method type is not yet supported in the release build.